docs/getting-started.md

```diff
@@ -21,6 +21,26 @@ The Client SDK is a set of user-facing programs to interact with Ledger and Audi
 ScalarDL (Ledger and Auditor) abstracts data as a set of assets, where each asset is composed of the history of a record identified by a key called `asset_id` and a historical version number called `age`.
 In this document, you will create a very simple application to manage an asset's status using ScalarDL Client SDK.
 
+## Install a JDK
+
+Because ScalarDL is written in Java, using Java is one of the easist ways to build your ScalarDL applications.
+In such a case, you must have one of the following Java Development Kits (JDKs) installed in your environment:
+
+- [Oracle JDK](https://www.oracle.com/java/technologies/downloads/) LTS version (8, 11, or 17)
+- [OpenJDK](https://openjdk.org/install/) LTS version (8, 11, or 17)
+
+Also, since ScalarDL is built with JDK 8, you must use JDK 8 to build your contracts.
+If you use JDK versions other than 8, specify `-release 8` option for `javac` or define what toolchain to use in your Gradle and Maven configurations.
+For the Gradle and Maven configurations, check [the Gradle doc](https://docs.gradle.org/current/userguide/toolchains.html) and [the Maven doc](https://maven.apache.org/guides/mini/guide-using-toolchains.html).
+
+{% capture notice--info %}
+**Note**
+
+We recommend using the LTS versions mentioned above, but other non-LTS versions may work.
```

Comment:
From the product/technical perspective, I think there is no problem with this description.
However, from the support perspective, I feel `may work` is a bit ambiguous.
Sometimes, I get an inquiry such as `What Java version do you support?` from a customer on the support site.

So, I want to define whether we will support the non-LTS version or not in the support site for users who have a commercial license.

If we don't support the not-LTS version, it might be better to add a note like `We don't provide technical support for non-LTS use cases even if you have a commercial license.`.

If such a description is not appropriate in this document (since this document is a technical document), I want to discuss it in a support team meeting.

@feeblefakie @NatsukiMiyamoto 
What do you think?

Reply:
@kota2and3kan  Let's talk separately. I agree your points that this may be a bit dangerous expression. 
However, I want to avoid to write "don't provide the technical support" in the product documentation as much as possible. (As far as I know, the product doc says what the product can do and cannot do, and then this means we can support and cannot support.) 

I suppose: 
- **This may work** means we want to say it may work but it is not supported and tested. 
- The users may take it may work , should work without testing, but Scalar should support since the doc says it work.   (In the technical support world, the users always take positive to their side)

My suggestion is just describe what we support like  "We recommend using LTS versions mentioned above as the only supported JDK." or something like that. 
(We don't want to discuss about non-supported environment may/should work in the doc or not and if we describe it clearly, then this provide the negative support to the customer. FAQ or some other place can follow this up if possible, as it's true this may work technically. ) 

@josh-wong please advise me how to mention non-supported scenario in the doc?

Reply:
@NatsukiMiyamoto  
OK. Let's discuss it in the support meeting! 🙇‍♀️
(I think we need to update both ScalarDB/ScalarDL documents.)

Reply:
@NatsukiMiyamoto @kota2and3kan 
> @josh-wong please advise me how to mention non-supported scenario in the doc?

Below or a couple suggestions. I know these suggestions might seem a bit strongly worded, but I think we should be accurate rather than vague or unclear.

```suggestion
We recommend using the LTS versions mentioned above since they are the only JDK versions that we support.
```

Or maybe the following?:

```suggestion
We recommend using the LTS versions mentioned above since we have tested operability on only those JDK versions.
```

Reply:
Thank you Josh. I prefer the first one. 
(I prefer to specify the supported version regardless tested or not.)

Reply:
@NatsukiMiyamoto Totally agree👍

Reply:
@josh-wong  
Thank you for your suggestion!
We decided to discuss it in the support meeting since we should apply this update to the ScalarDB docs side too.
So, in this PR, we don't apply this update.
We will do it in another PR after additional discussion.

@NatsukiMiyamoto  
As we mentioned in the support daily meeting, let's discuss it in the next support weekly meeting!

Reply:
@kota2and3kan @NatsukiMiyamoto So, I'm not applying the suggestion. Thanks!